app/services/oa_workflow_service.rb

```diff
@@ -33,5 +33,14 @@ def workflow
     rescue StandardError
       file.update_column(:downloaded, false)
     end
+
+    ActivityInsightOAFile.needs_version_check.each do |file|
+      file.version_checked = true
+      file.save!
+      AiOAWfVersionCheckJob.perform_later(file.id)
+    rescue StandardError
+      file.update_column(:version_checked, false)
```

Comment:
Should we add the `version_checked` attribute to the RailsAdmin list view of the files so that we can filter for the files that are in this state?

Reply:
Yeah, that is a good idea.  I'll do that.